algorithms/HPdclassifier/R/hpdrpart.R

```diff
@@ -0,0 +1,279 @@
+#####################################################################################
+# Copyright [2013] Hewlett-Packard Development Company, L.P.                        # 
+#                                                                                   #
+# This program is free software; you can redistribute it and/or                     #
+# modify it under the terms of the GNU General Public License                       #
+# as published by the Free Software Foundation; either version 2                    #
+# of the License, or (at your option) any later version.                            #
+#                                                                                   #
+# This program is distributed in the hope that it will be useful,                   #
+# but WITHOUT ANY WARRANTY; without even the implied warranty of                    #
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                      #
+# GNU General Public License for more details.                                      #
+#                                                                                   #
+# You should have received a copy of the GNU General Public License                 #
+# along with this program; if not, write to the Free Software                       #
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.    #
+#####################################################################################
+
+
+hpdrpart <- function(formula, data, weights, subset , na.action = na.omit, 
+	 model = FALSE, x = FALSE, y = FALSE, params = NULL, 
+	 control = NULL, cost = NULL, 
+	 completeModel = TRUE, nBins = 256L, do.trace = FALSE)
```

Comment:
So far, except in the hpdRF_parallelTree algorithm, we have used "trace" instead of "do.trace" everywhere in the functions of platform and algorithms. I know that "do.trace" initially comes from randomForest package, and I think we'd better not change the interface of hpdRF_parallelTree. Nevertheless, it might be better if use "trace" after this for the sake of consistency.